09/style.css

```diff
@@ -0,0 +1,38 @@
+.container {
+display: inline-block;
+}
+
+.triangle-up {
+    width: 0px;
+    height: 0px;
+    border-bottom: 100px solid black;
+    border-left: 100px solid transparent;
+    border-right: 0px solid transparent;
+}
+.triangle-down {
+    width: 0px;
+    height: 0px;
+    border-top: 100px solid black;
+    border-right: 50px solid transparent;
+    border-left: 50px solid transparent;
+}
+
+.triangle-right {
+    width: 150px;
+    height: 80px;
+    background: #444;
+    border-radius: 5px;
+    position: relative;
```

Comment:
Position must be written on the top.
Please read this article https://webformyself.com/eshhe-raz-poryadok-svojstv-css/